samples/uss-profile-sample/src/SshUssApi.ts

```diff
@@ -58,6 +58,10 @@ export class SshUssApi implements MainframeInteraction.IUss {
         });
     }
 
+    public uploadFromBuffer(buffer: Buffer, filePath: string, options?: IUploadOptions): Promise<IZosFilesResponse> {
```

Comment:
If `uploadFromBuffer` is required, will we need to implement this for the USS profile sample to work with v3?

Reply:
Yes - I don't mind adding one now if we want the sample to be ready ahead of time for the FileSystemProvider changes.

Otherwise, I planned on providing an implementation (as well as a utility function for creating a temp file for upload) in the FileSystemProvider PR 😋

Reply:
Added in [f07139f](https://github.com/zowe/vscode-extension-for-zowe/pull/2738/commits/f07139fb8f73f9536edd1ec72bc35aeee5010df8)